cmake/modules/FindLibSndFile.cmake

```diff
@@ -19,16 +19,14 @@ find_path(LIBSNDFILE_INCLUDE_DIR
 
 # Library
 find_library(LIBSNDFILE_LIBRARY
-	NAMES sndfile libsndfile-1
+	NAMES sndfile sndfile-1
 	PATHS ${LIBSNDFILE_PKGCONF_LIBRARY_DIRS}
 )
 
-find_package(PackageHandleStandardArgs)
-find_package_handle_standard_args(LibSndFile  DEFAULT_MSG  LIBSNDFILE_LIBRARY LIBSNDFILE_INCLUDE_DIR)
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(LibSndFile DEFAULT_MSG LIBSNDFILE_LIBRARY LIBSNDFILE_INCLUDE_DIR)
 
-if(LIBSNDFILE_FOUND)
-  set(LIBSNDFILE_LIBRARIES ${LIBSNDFILE_LIBRARY})
-  set(LIBSNDFILE_INCLUDE_DIRS ${LIBSNDFILE_INCLUDE_DIR})
-endif(LIBSNDFILE_FOUND)
+set(LIBSNDFILE_LIBRARIES ${LIBSNDFILE_LIBRARY})
+set(LIBSNDFILE_INCLUDE_DIRS ${LIBSNDFILE_INCLUDE_DIR})
```

Comment:
Why unconditionally? imo this should still depend on the library being found - that's kind of cmake standard / expected behaviour?